cmake/compilerFlags.cmake

```diff
@@ -90,6 +89,10 @@ if ( MINGW OR UNIX OR MSYS ) # MINGW, Linux, APPLE, CYGWIN
             endif()
         endif()
     endif()
+    if ( EXIV2_BUILD_USE_C++11 ) 
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-deprecated")

```

Comment:
Please, replace the usage of `CMAKE_CXX_FLAGS` by the more appropriate usage of `add_compile_options`:

```
  add_compile_options(-Wno-deprecated)
```

Reply:
I've removed that 100%.  I don't mess with the compile_options for `-Wno-deprecated`  The user can pass that from the command when required.  And that's what I've documented in README.md.